src/core/server/logging/appenders/rewrite/rewrite_appender.ts

```diff
@@ -52,7 +52,7 @@ export class RewriteAppender implements DisposableAppender {
    * @param record `LogRecord` instance to be logged.
    */
   public append(record: LogRecord) {
-    const transformedRecord = this.policy.transform(record);
+    const rewriteedRecord = this.policy.rewrite(record);
```

Comment:
nit: `rewriteedRecord` -> `rewrittenRecord`

Reply:
Oops, that was a messed up grep. Thanks for catching 🙂